ui/package.json

```diff
@@ -49,6 +49,7 @@
     "protobufjs": "^7.2.5",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-select": "^5.8.0",
```

Comment:
Why is this necessary given existing dependency on @radix-ui/react-select ?

Reply:
We want the select dropdowns to be searchable - which radix ui's react select does not seem to have

Reply:
A separate PR should be made making the switch, removing @radix-ui/react-select too